lib/icingadb/icingadb.cpp

```diff
@@ -52,9 +57,45 @@ void IcingaDB::Start(bool runtimeCreated)
 {
 	ObjectImpl<IcingaDB>::Start(runtimeCreated);
 
-	boost::call_once([]() {
-		m_EnvironmentId = SHA1(GetEnvironment());
-	}, m_EnvironmentIdOnce);
+	std::call_once(m_EnvironmentIdOnce, []() {
```

Comment:
Less boost dependent.

Reply:
The main motivation for the change was that Google finds the documentation for an [ancient version](https://www.boost.org/doc/libs/1_33_1/doc/html/call_once.html) when searching for "boost::call_once" that states "Requires: The function func shall not throw exceptions."

But when looking at this again, (at least) Boost 1.66+ seem to also [handle this just fine](https://www.boost.org/doc/libs/1_66_0/doc/html/thread/synchronization.html#thread.synchronization.once.call_once).

`std::call_once` also [handles exceptions properly](https://en.cppreference.com/w/cpp/thread/call_once), so I'd still go with the standard version, because why not.